### DISCUSSION TITLE
Koboldcpp REST API

### DISCUSSION THREAD

**Discussion by evstarshov** [LostRuins/koboldcpp#143] [Q&A] [answered]
Does this project has REST API? I want to try to link my server running koboldcpp with a 3rd party application. I tried to open http://localhost:5001/api, but it's wrong url. Is there any docs or maybe swagger where I can explore the methods?

Thank you!

**Comment by LostRuins** [comment, marked as answer]
It has a subset of the Kobold REST API. Check the /generate function.

https://link.concedo.workers.dev/koboldapi
